### PR TITLE
No console output in tests

### PR DESCRIPTION
**Pull request by daytonn**

### CODE REVIEW

File: karma.devicelab.js [hunk 1]
```diff
@@ -87,18 +87,7 @@ karmaConfig.customLaunchers = {
...
-  //'bs_android_4_3',
-  //'bs_android_4_4',
-  //'bs_android_5_0',
-  //'bs_ios_8_3',
-  //'bs_ios_9_1',
-  //'bs_ie_10',
-  //'bs_ie_11',
-  //'bs_safari_8',
   'bs_chrome_47',
-  //'bs_firefox_43',
```
Comment (collin) [inline karma.devicelab.js]: Alright, I think what we actually want to do here, is set up a grid in our `.travis.yml` that passes these browsers in as an environment variable. This way we get better reporting about which device is failing.

Comment (daytonn) [inline karma.devicelab.js]: Can you set that up? I'm not exactly sure what needs to be done.

Comment (collin) [inline karma.devicelab.js]: Yeah, I'll keep that on my plate. 👍  for now.

**Comment by kand** [comment]
👍

**Pull request merged**